requirements/development.txt

```diff
@@ -6,3 +6,7 @@ pytest
 pytest-django
 pytest-asyncio
 pytest-cov
+Werkzeug
+pyparsing
+pydot
+django-extensions
```

Comment:
Please explain why we need all of this new requirements

Reply:
The explanation to this is in my comment before.